src/NServiceBus.Transport.SQS/InputQueuePump.cs

```diff
@@ -41,10 +42,32 @@ class InputQueuePump : IMessageReceiver
             Subscriptions = subscriptionManager;
 
             this.coreSettings = coreSettings;
+            this.setupInfrastructure = setupInfrastructure;
         }
 
         public async Task Initialize(PushRuntimeSettings limitations, OnMessage onMessage, OnError onError, CancellationToken cancellationToken = default)
         {
+            try
+            {
+                inputQueueUrl = await queueCache.GetQueueUrl(ReceiveAddress, cancellationToken)
+                .ConfigureAwait(false);
+            }
+            catch (QueueDoesNotExistException ex)
+            {
+                var msg = $"Queue `{ReceiveAddress}` doesn't exist. Call endpointConfiguration.EnableInstallers() to create the queues at startup, or create them manually.";
```

Comment:
Mauro: We can probably avoid creating the string based on the setup infrastructure flag